libs/sfm/bundler_incremental.cc

```diff
@@ -188,6 +190,41 @@ Incremental::reconstruct_next_view (int view_id)
     return true;
 }
 
+void
+Incremental::try_restore_tracks_for_views (void)
+{
+  for (std::size_t i = 0; i < this->viewports->size(); ++i)
```

Comment:
Did you use two spaces for indentation on purpose?

Reply:
whoops :D